packages/flutter/lib/src/rendering/list_wheel_viewport.dart

```diff
@@ -243,6 +247,47 @@ class RenderListWheelViewport
     _hasScrolled();
   }
 
+  /// Visual effect based on how we see the wheel. Imagine we are at some
+  /// distance away from the screen. The wheel is at the center of the screen.
+  /// If we move our position in the xAxis, we will see the wheel differently.
+  /// -----------
+  /// '         '
+  /// '         '
+  /// A    B    C
+  /// '         '
+  /// '         '
+  /// -----------
+  /// By default, we are at point B (the center). If we are at point A, the
+  /// wheel will curve to the left a little, for example. Similar for point C.
+  /// The centerViewRate is defaulted at 1.0, which is point B. 0.0 will be point A
+  /// and 2.0 will be point C. There's no limit to centerViewRate since we can be
+  /// infinite to the left or right, but a reasonable value would be in the
+  /// range [0.0, 2.0].
+  double get centerViewRate => _centerViewRate;
+  double _centerViewRate = 1.0;
+  set centerViewRate(double value) {
+    if(value == null) {
+      _centerViewRate = 1.0;
+      return;
+    }
+    _centerViewRate = value;
```

Comment:
you need to either call markNeedsLayout or markNeedsPaint, otherwise this render object wouldn't update.